test/shared/src/main/scala/zio/test/NewOrLegacySpec.scala

```diff
@@ -0,0 +1,6 @@
+package zio.test
+
+sealed trait NewOrLegacySpec
```

Comment:
Does this need to be in ZIO Test or can it be moved to ZIO Test SBT?

Reply:
Another good catch- moved!